src/components/date-picker/date-picker.tsx

```diff
@@ -193,7 +200,7 @@ export class DatePicker {
   // --------------------------------------------------------------------------
   connectedCallback(): void {
     if (Array.isArray(this.value)) {
-      this.valueAsDate = this.value.map((v) => dateFromISO(v));
+      this.valueAsDate = mapValueToValueAsDate(this.value);
```

Comment:
nitpick: I'm not sold on the name of this utility function.

I actually think it was more readable before :D

Reply:
How about `getValueAsDate` ?

Reply:
Maybe `getValueAsDateRange`